src/examples/component-definition/xml/example-component-definition.xml

```diff
@@ -26,6 +26,9 @@
     <responsible-role role-id="provider">
       <party-uuid>ef7c799a-c50e-49ab-83e0-515e989e6df1</party-uuid>
     </responsible-role>
+    <responsible-role role-id="customer">
+      <party-uuid>ef7c799a-c50e-49ab-83e0-515e982305a0</party-uuid>
+    </responsible-role>
```

Comment:
You do not want to bind the MongoDB organization to the customer role for the component in the CDEF as that customer may not be known. We should stub it without a specific party identified by UUID.

```suggestion
    <responsible-role role-id="customer"/>
```

Reply:
You are right. I did not pay attention. This was done 2 years ago and if I recall, I only added the parent controls. BUT - my fault for not catching it at that time. I agree with eh change.

Reply:
No worries . We committed the change in one place not this one as well. I will unresolve this comment and rec, add it in, re-resolve this conversation topic, approve, and merge.

Reply:
Thank you. Good to go.